doc/changelog.rst

```diff
@@ -1,6 +1,36 @@
 Changelog
 =========
 
+Changes in Version 4.3
+----------------------
+
+PyMongo 4.3 brings a number of improvements including:
+
+- Improved support for documents with BSON datetimes outside of the range
+  supported by Python. See :ref:`handling-out-of-range-datetimes` for examples.
+- Improved support for avoiding deadlocking when a ``fork()`` occurs with an
+  :class:`~pymongo.mongo_client.MongoClient`.
+- Refactored several underlying classes in :mod:`pymongo.monitoring` for
+  connection events.
+
+Bug fixes
+.........
+
+- Updated :class:`~pymongo.change_stream.ChangeStream` so that it will
+  close after non-resumable non-timeout errors, and will resume its connection
+  otherwise.
```

Comment:
I'd say "Fixed a bug where ChangeStream would allow an app to retry calling next() or try_next() even after non-resumable errors (`PYTHON-3389`_)"